Source/ACE/Entity/Creature.cs

```diff
@@ -495,5 +497,162 @@ protected virtual void UpdateVitalInternal(CreatureVital vital, uint newVal)
                 new ActionChain(this, () => VitalTickInternal(vital)).EnqueueChain();
             }
         }
+
+        public override void SerializeIdentifyObjectResponse(BinaryWriter writer)
```

Comment:
I did no refactoring, just trying to see if this is generally what you were saying